### ISSUE TITLE
Defining Research Questions

### ISSUE THREAD

**Issue by bzrudski** [QLS600/assignment#5]
In this thread, we will work on defining questions to answer during the assignment in order to assess and analyse code and data availability.

**Comment by bzrudski** [comment]
Potential comparison: open access vs. non-open access. Are the data and code made available more often when the paper is published in an open access journal?

**Comment by bzrudski** [comment]
Also, should we have a separate category for papers that include summary data? Or should we only concentrate on full datasets?

**Comment by aosakwe** [comment]
I think we need a separate category. The summary data is nice to have but it is not as valuable as actually sharing the full dataset.
Other questions:
How accessible is the code and data from the top publishing authors?
Is year of publication a good predictor of data/code availability?

**Comment by bzrudski** [comment]
> Is year of publication a good predictor of data/code availability?

Good idea! I think we should look separately at code and at data. That way, if there is a change, we could determine if code availability increased before or after data availability. I also really like the idea of tying it back into the publishing record of the authors.

**Comment by bzrudski** [comment]
Based on the above thread, these seem to be our questions (assuming we keep code and data separate; otherwise we can just combine the two):
* Is year of publication a good predictor of data availability?
* Is year of publication a good predictor of code availability?
* Are data from top-publishing authors accessible?
* Is code from top-publishing authors accessible?
* Are data more likely to be available if the paper is published in an open access journal?
* Is the code more likely to be available if the paper is published in an open access journal?

Thanks @aosakwe for your suggestions! Next, we need to figure out how to quantitatively answer these questions.

**Comment by aosakwe** [comment]
Sounds good to me @bzrudski ! We can go over this with everyone in class tomorrow to wrap up what tasks people want to do and how much we can finish for Thursday.
